web/scenes/Portal/Teams/TeamId/Apps/page/ClientPage/index.tsx

```diff
@@ -15,7 +15,7 @@ export const ClientPage = () => {
 
   return (
     <Fragment>
-      <SizingWrapper gridClassName="h-full flex justify-center items-center -mt-10">
+      <SizingWrapper gridClassName="h-full flex justify-center items-center">
```

Comment:
```suggestion
      <SizingWrapper gridClassName="h-full flex justify-center items-center pb-10">
```

I added this originally since the content sat a bit low on the page in my opinion. This should fix boundary issues?